linkerd/distribute/src/params.rs

```diff
@@ -0,0 +1,80 @@
+use rand::distributions::{WeightedError, WeightedIndex};
+use std::sync::Arc;
+
+/// A parameter type that configures how a [`Distribute`] should behave.
```

Comment:
tiny nit: docs link is broken
```suggestion
/// A parameter type that configures how a [`Distribute`] should behave.
///
/// [`Distribute`]: crate::service::Distribute
```